dev/com.ibm.websphere.appserver.features/visibility/protected/com.ibm.wsspi.appserver.webBundle-1.0.feature

```diff
@@ -4,11 +4,9 @@ symbolicName = com.ibm.wsspi.appserver.webBundle-1.0
 WLP-DisableAllFeatures-OnConflict: false
 visibility = protected
 
--bundles= \
- com.ibm.ws.eba.wab.integrator
-
 -features=com.ibm.websphere.appserver.servlet-3.0; ibm.tolerates:="3.1,4.0,5.0,6.0", \
-  io.openliberty.webBundle.internal.ee-6.0; ibm.tolerates:="9.0"
+  io.openliberty.servlet.internal-3.0; ibm.tolerates:="3.1,4.0,5.0,6.0", \
```

Comment:
I'm not following why this inclusion is needed when `com.ibm.websphere.appserver.servlet` already is included.  Doesn't `io.openliberty.servlet.internal` just redirect directly to `com.ibm.websphere.appserver.servlet`?

If the idea is that `io.openliberty.servlet.internal` is used to avoid tolerates propagation (because it is internal).  Then this is about having auto features being enabled based off `io.openliberty.servlet.internal` being enabled?

Is the idea that any of our own features that use to include `com.ibm.websphere.appserver.servlet` directly now include `io.openliberty.servlet.internal` instead for 5.0 and 6.0?  Then all our auto features can depend on being enabled with `io.openliberty.servlet.internal`?  What about user features that depend on `com.ibm.websphere.appserver.servlet-5.0`.  Does that still cause all our auto-features to be enabled that were before?

Reply:
The 3.0, 3.1, 4.0 and 5.0 internal servlet feature includes the public servlet feature, but with 6.0 the public servlet feature includes the internal servlet feature.  This allows for us to not enable the public feature for 6.0.  The public servlet feature is needed here since this is the protected feature and the servlet API is requires to be made available to the user.  

Since the internal brings in the public feature for 3.0, 3.1, 4.0 and 5.0, the auto features for those are still going to be satisfied.  For 6.0 the auto features reference the internal servlet feature since that is the one that is enabled whether using the private or the internal feature.

Reply:
I just created the internal servlet 5.0 feature for testing in order to test EE9 features without the public servlet feature for some features before the features had new tests for EE10 so I could know to add the public servlet feature to the server.xml to help people accelerate EE10 repeats.  No one uses the feature in their feature because that would be a zero migration breaking change.